tests/test_appdataclass.py

```diff
@@ -85,3 +85,38 @@ def test_find_dependencies_to_install(monkeypatch, installed_packages, python_bi
 
     assert "aiida-core" not in dependencies_name
     assert "jupyter-client" in dependencies_name
+
+
+def test_update_status_issue_360(monkeypatch, installed_packages, python_bin):
```

Comment:
I am not a fan of referring to a specific github issue as a test name.

```suggestion
def test_update_status_latest_version_incompatible(monkeypatch, installed_packages, python_bin):
```